docs/bn/week06/06-3.md

```diff
@@ -0,0 +1,398 @@
+
+$$
+W_{hx}\cdot x[t]+W_{hh}\cdot h[t-1]
+$$
+
+<!--which corresponds to the stack representation of the input.-->
+যা ইনপুটটির স্ট্যাক উপস্থাপনার সাথে মিলে যায়।
+
+
+<!--$y[t]$ is calculated at the final rotation and then we can use the chain rule to backpropagate the error to the previous time step.-->
+$y[t]$ চূড়ান্ত রোটেশনের সময় গণনা করা হয় এবং তারপরে আমরা ত্রুটিটিকে আগের সময়ের ধাপে ব্যাকপ্রোপাগেট করতে চেইন বিধিটি ব্যবহার করতে পারি।
+
+<!--### Batch-Ification in Language Modeling-->
+### ভাষা মডেলিংয়ে ব্যাচ-ইফিকেশন
+
+<!--When dealing with a sequence of symbols, we can batchify the text into different sizes. For example, when dealing with sequences shown in the following figure, batch-ification can be applied first, where the time domain is preserved vertically. In this case, the batch size is set to 4.-->
+প্রতীকগুলির ক্রম নিয়ে কাজ করার সময়, আমরা লেখ্যটি বিভিন্ন আকারে দলভুক্ত করতে পারি। উদাহরণস্বরূপ, নিম্নলিখিত চিত্রটিতে প্রদর্শিত ক্রমগুলির সাথে কাজ করার সময়, ব্যাচ-ইফিকেশন প্রথমে প্রয়োগ করা যেতে পারে, যেখানে সময় ডোমেনটি উল্লম্বভাবে সংরক্ষণ করা হয়। এই ক্ষেত্রে, ব্যাচের আকার ৪  এ সেট করা আছে।
+<center>
+<img src="{{site.baseurl}}/images/week06/06-3/batchify_1.png" style="zoom: 40%; background-color:#DCDCDC;"/><br>
+<!--<b>Figure 18:</b> Batch-Ification-->
+চিত্র ১৮ঃ ব্যাচ-ইফিকেশন
+</center>
+
+<!--If BPTT period $T$ is set to 3, the first input $x[1:T]$ and output $y[1:T]$ for RNN is determined as-->
+বিপিটিটি পিরিয়ড $টি$ ৩ এ সেট করা থাকলে, আরএনএন-এর জন্য প্রথম ইনপুট $x [1: T]$ এবং আউটপুট $y[1: T]$  হিসাবে নির্ধারিত হয়।
+$$
+\begin{aligned}
+x[1:T] &= \begin{bmatrix}
+a & g & m & s \\
+b & h & n & t \\
+c & i & o & u \\
+\end{bmatrix} \\
+y[1:T] &= \begin{bmatrix}
+b & h & n & t \\
+c & i & o & u \\
+d & j & p & v
+\end{bmatrix}
+\end{aligned}
+$$
+
+<!--When performing RNN on the first batch, firstly, we feed $x[1] = [a\ g\ m\ s]$ into RNN and force the output to be $y[1] = [b\ h\ n\ t]$. The hidden representation $h[1]$ will be sent forward into next time step to help the RNN predict $y[2]$ from $x[2]$. After sending $h[T-1]$ to the final set of $x[T]$ and $y[T]$, we cut gradient propagation process for both $h[T]$ and $h[0]$ so that gradients will not propagate infinitely(.detach() in Pytorch). The whole process is shown in figure below.-->
+প্রথম ব্যাচে আরএনএন করার সময়, প্রথমত, আমরা আরএনএন-তে $x[1] = [a\ g\ m\s]$  ব্যবহার করি এবং আউটপুটকে $y [1] = [b\ h\n\ t]$  হতে বাধ্য করি। $x[2]$  থেকে $y[2]$ কে  আরএনএন দ্বারা  পূর্বাভাসে  সহায়তা করার জন্য হিডেন উপস্থাপনা $h[1]$ টিকে   পরবর্তী সময় ধাপে প্রেরণ করা হবে।  $x[T]$ এবং $y[T]$ এর চূড়ান্ত সেটে $h[T-1]$ প্রেরণের পরে, আমরা $ h[T]$ এবং $h[0]$ উভয়ের জন্য গ্র্যাডিয়েন্ট প্রোপাগেশন  প্রক্রিয়াটি কেটে দিলাম যাতে করে গ্র্যাডিয়েন্ট গুলো  অসম্পূর্ণভাবে প্রোপাগেট করবে না (.detach () in Pytorch) । পুরো প্রক্রিয়াটি নীচের চিত্রে দেখানো হয়েছে।
```

Comment:
More broken math?
Please, render your contribution and make sure it actually renders properly.

Reply:
Rendered and checked, I hope it's correct now.